cmd/collector/main.go

```diff
@@ -142,6 +143,7 @@ func main() {
 	command.AddCommand(env.Command())
 	command.AddCommand(docs.Command(v))
 	command.AddCommand(status.Command(v, ports.CollectorAdminHTTP))
+	command.AddCommand(printconfig.Command(v))
```

Comment:
There are other binaries in Jaeger, add this to all of them

Reply:
I've also added the following binaries: all-in-one, collector, ingester, jaeger, query and remote-storage. Are there any other binaries I could add?